lnd_methods/offchain/get_pending_payments.js

```diff
@@ -0,0 +1,190 @@
+              base_fee_mtokens: <Base Fee Millitokens String>
+              cltv_delta: <Locktime Delta Number>
+              fee_rate: <Fees Charged in Millitokens Per Million Number>
+              [is_disabled]: <Channel is Disabled Bool>
+              max_htlc_mtokens: <Maximum HLTC Millitokens Value String>
+              min_htlc_mtokens: <Minimum HTLC Millitokens Value String>
+              updated_at: <Updated At ISO 8601 Date String>
+            }
+            [timeout_height]: <Error CLTV Timeout Height Number>
+            [update]: {
+              chain: <Chain Id Hex String>
+              channel_flags: <Channel Flags Number>
+              extra_opaque_data: <Extra Opaque Data Hex String>
+              message_flags: <Message Flags Number>
+              signature: <Channel Update Signature Hex String>
+            }
+          }
+          message: <Error Message String>
+        }
+        [index]: <Payment Add Index Number>
+        [confirmed_at]: <Payment Confirmed At ISO 8601 Date String>
+        is_confirmed: <Payment Attempt Succeeded Bool>
+        is_failed: <Payment Attempt Failed Bool>
+        is_pending: <Payment Attempt is Waiting For Resolution Bool>
+        route: {
+          fee: <Route Fee Tokens Number>
+          fee_mtokens: <Route Fee Millitokens String>
+          hops: [{
+            channel: <Standard Format Channel Id String>
+            channel_capacity: <Channel Capacity Tokens Number>
+            fee: <Fee Number>
+            fee_mtokens: <Fee Millitokens String>
+            forward: <Forward Tokens Number>
+            forward_mtokens: <Forward Millitokens String>
+            [public_key]: <Forward Edge Public Key Hex String>
+            [timeout]: <Timeout Block Height Number>
+          }]
+          mtokens: <Total Fee-Inclusive Millitokens String>
+          [payment]: <Payment Identifier Hex String>
+          timeout: <Timeout Block Height Number>
+          tokens: <Total Fee-Inclusive Tokens Number>
+          [total_mtokens]: <Total Millitokens String>
+        }
+      }]
+      created_at: <Payment at ISO-8601 Date String>
+      [destination]: <Destination Node Public Key Hex String>
+      id: <Payment Preimage Hash String>
+      [index]: <Payment Add Index Number>
+      is_confirmed: <Payment is Confirmed Bool>
+      is_outgoing: <Transaction Is Outgoing Bool>
+      mtokens: <Millitokens Attempted to Pay to Destination String>
+      [request]: <BOLT 11 Payment Request String>
+      safe_tokens: <Payment Tokens Attempted to Pay Rounded Up Number>
+      tokens: <Rounded Down Tokens Attempted to Pay to Destination Number>
+    }]
+    [next]: <Next Opaque Paging Token String>
+  }
+*/
+module.exports = ({limit, lnd, token}, cbk) => {
+  return new Promise((resolve, reject) => {
+    return asyncAuto({
+      // Check arguments
+      validate: cbk => {
+        if (!!limit && !!token) {
+          return cbk([400, 'ExpectedNoLimitWhenPagingPayFailuresWithToken']);
```

Comment:
the error code here should reference PendingPayments not PayFailures

Reply:
fixed https://github.com/alexbosworth/lightning/pull/63/commits/779cea3a512c4c207530cbc16eff042604807c45